Auth0Tests/CredentialsSpec.swift

```diff
@@ -39,8 +39,7 @@ class CredentialsSpec: QuickSpec {
         describe("init from json") {
 
             it("should have all tokens and token_type") {
-                let credentials = Credentials(json: ["access_token": AccessToken, "token_type": Bearer, "id_token": IdToken, "refresh_token": RefreshToken, "expires_in" : expiresIn, "scope": Scope, "recovery_code": RecoveryCode])
-                expect(credentials).toNot(beNil())
```

Comment:
When created from JSON, the instance of `Credentials` cannot be `nil`. That assertion is superflous.